fili-core/src/main/java/com/yahoo/bard/webservice/table/BasePhysicalTable.java

```diff
@@ -96,6 +97,22 @@ public DateTime getTableAlignment() {
 
     @Override
     public SimplifiedIntervalList getAvailableIntervals(DataSourceConstraint constraint) {
+
+        Set<String> tableColumnNames = getSchema().getColumnNames();
+        Set<String> invalidColumnNames = constraint.getAllColumnNames().stream()
+                .filter(name -> !tableColumnNames.contains(name))
+                .collect(Collectors.toSet());
+
+        if (!constraint.getAllColumnNames().stream().allMatch(tableColumnNames::contains)) {
+            String message = String.format(
+                    "Received invalid request requesting for columns: %s that is not available in this table: %s",
+                    invalidColumnNames.stream().collect(Collectors.joining(",")),
```

Comment:
Rather than computing `invalidColumnNames` every time, let's replace the formatting stream here with the computation for the invalid names:

```java
constraint.getAllColumnNames().stream().filter(name -> !tableColumnNames.contains(name)).collect(Collectors.joining(","))
```